scene/gui/tabs.cpp

```diff
@@ -295,6 +299,7 @@ void Tabs::_notification(int p_what) {
 				}
 
 				Ref<StyleBox> sb;
+				Ref<Font> fo;
```

Comment:
Using `sb` for `StyleBox` on the above line is a bad convention, so you should not try to emulate it. `font` would have been just fine as identifier.